src/controllers/customContentController.js

```diff
@@ -1,28 +1,30 @@
-const Handlebars = require('handlebars');
+/* eslint-disable no-undef */
 const { renderTemplate, renderTemplateFromAPI, loadMarkdown } = require('../utils/util');
-const config = require('../config/config');
+const config = require(process.cwd() + '/config.json');
 const markdown = require('marked');
 const fs = require('fs');
 const path = require('path');
+const adminDao = require('../dao/admin');
+const constants = require('../utils/constants');
 
+const filePrefix = config.pathToContent;
 
 const loadCustomContent = async (req, res) => {
-
+    
     let html = "";
     const orgName = req.originalUrl.split("/")[1];
     let filePath = req.originalUrl.split("/" + orgName + "/").pop();
-    let baseURL = orgName
-    if (config.mode == 'single' || config.mode == 'design') {
+    console.log("CUSTOM")
+
+    console.log(req.originalUrl)
+    if (config.mode === constants.DEV_MODE) {
 
         let templateContent = {};
-        if (config.mode == 'design') {
-            baseURL = "http://localhost:" + config.port;
-            filePath = req.originalUrl.split(baseURL).pop();
-        }
-        templateContent["baseUrl"] = baseURL
+        templateContent["baseUrl"] = constants.BASE_URL + config.port
```

Comment:
In some of the places the trailing semicolon is missing.

Reply:
added